src/index.css

```
@@ -104,6 +105,70 @@ h1 {
   margin-top: 8px;
 }
 
+nav {
+  background-color: #262626;
+  width: 100%;
+  position: sticky;
+  top: 0;
+  height: auto;
+}
+
+nav li {
+  font-size: 19px;
+  font-weight: 700;
+  line-height: 26px;
+  text-align: center;
+  margin-right: 16px;
+  margin-bottom: 8px;
+  margin-left: 16px;
+  padding: 8px 16px 8px 16px;
```

If you have the same value for top and bottom and the same value for right and left, then you only need to specify two values:

```suggestion
  padding: 8px 16px;
```

Maybe you're already aware of it but if not, [here's the explanation from MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/padding#syntax):

> * When **one** value is specified, it applies the same padding to **all four sides**.
> * When **two** values are specified, the first padding applies to the **top and bottom**, the second to the **left and right**.
> * When **three** values are specified, the first padding applies to the **top**, the second to the **right and left**, the third to the **bottom**.
> * When **four** values are specified, the paddings apply to the **top**, **right**, **bottom**, and **left** in that order (clockwise).